module/dkg/utils.go

```diff
@@ -0,0 +1,22 @@
+package dkg
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func GetDKGCommitteeIndex(nodeID flow.Identifier, participants flow.IdentityList) (int, error) {
+	myIndex := -1
```

Comment:
Nit: the function here is now more general (find _any_ node's index), but the variable name still reflects the more specific use the code had previously (find the local node's index).
```suggestion
	nodeIndex := -1
```

Reply:
https://github.com/onflow/flow-go/pull/1849/commits/aa369fc7718a37db5db1e76cb77481bc0c1cca33